week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -6,18 +6,23 @@
   They have also decided that they should add a 1% fee to all foreign transactions
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
-
-function convertToUSD(price) {}
-function convertToBRL(price) {}
-
+function transactionFee(price) {
+	return price * 0.01;
+}
+function convertToUSD(price) {
+	return price * 1.4 + transactionFee(price);
```

Comment:
So close! You have definitely understood how to use functions to abstract important logic (DRY/Don't Repeat Yourself principle) and that is good to see.

There is a slight error here, because the result of `price * 1.4` is a number in USD but the `transactionFee(price)` is still in GBP. You need to also convert the transaction fee to the correct currency or it makes no sense to add them.

Do you want to try and re-arrange the maths to correct this?